### ISSUE TITLE
Unable to query for cases with TestRail 5.2 or later.

### ISSUE THREAD

**Issue by fouadz** [codepine/testrail-api-java-client#8]
TestRail introduced a new concept of Templates. The case query returns a new field called template_id.

```
Exception in thread "main" java.lang.IllegalArgumentException: Case field list configuration is possibly outdated since it does not contain custom field: template_id 
 at com.google.common.base.Preconditions.checkArgument(Preconditions.java:125)
 at com.codepine.api.testrail.internal.CaseModule$CaseDeserializer.deserialize(CaseModule.java:87) 
 at com.codepine.api.testrail.internal.CaseModule$CaseDeserializer.deserialize(CaseModule.java:65) 
 at  com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:1270)
 at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:865) 
 at  com.codepine.api.testrail.Request.execute(Request.java:150)
 at com.codepine.api.testrail.Request$execute.call(Unknown Source)
```

**Comment by humanium** [comment]
Could someone provide any estimates on this?

**Comment by kunal546** [comment]
I will work on the fix to make Case compatible with TestRail 5.2 and release it in a couple of days.

**Comment by Rathore4u** [comment]
Did this release happen  ?
